pkg/proxy/ipvs/proxier.go

```diff
@@ -1702,8 +1704,9 @@ func (proxier *Proxier) cleanLegacyService(atciveServices map[string]bool, curre
 
 	for _, addr := range unbindIPAddr.UnsortedList() {
 		err := proxier.netlinkHandle.UnbindAddress(addr, DefaultDummyDevice)
-		if err != nil {
-			glog.Errorf("Failed to unbind service from dummy interface, error: %v", err)
+		// Ignore no such address error when try to unbind address
+		if err != nil && err != unix.ENXIO {
```

Comment:
this breaks cross build https://k8s-gubernator.appspot.com/build/kubernetes-jenkins/pr-logs/pull/59158/pull-kubernetes-cross/6701/

Reply:
Sorry, but I assume it's fixed by #59243